moveit_setup_assistant/src/tools/moveit_config_data.cpp

```diff
@@ -526,15 +526,15 @@ std::string MoveItConfigData::getGazeboCompatibleURDF()
       auto joint_tag = doc_element->FirstChildElement("joint");
       if (!type_tag || !type_tag->GetText() || !joint_tag || !joint_tag->Attribute("name"))
         continue;  // ignore invalid tags
-      if (static_cast<std::string>(type_tag->GetText()) == "transmission_interface/SimpleTransmission")
+      if (std::string(type_tag->GetText()) == "transmission_interface/SimpleTransmission")
```

Comment:
@rhaschke Good one. This is more save.

Reply:
Semantically it is exactly the same. But, I consider the proposed version more readable.